ci/README.md

```diff
@@ -9,13 +9,68 @@ File [k8s-conformance-image-version](k8s-conformance-image-version) stores the
 version number of the K8s conformance container image we currently use to run
 tests.
 
+## Antrea test suite
+
+We run 4 different categories of tests as part of CI:
+ * **unit tests**: most Go packages for Antrea include some unit tests written
+   using the [Go `testing`] package. Unit tests typically rely on mock testing
+   to isolate the package being tested, and abstract platform-specific
+   functionality hidden behing interfaces. When adding a new package or
+   modifying an existing one, add the appropriate unit tests.
+ * **integration tests**: these tests are located under [test/integration] and
+   are also written using the [Go `testing`] package. Unlike unit tests, they
+   typically exercise multiple Go packages to ensure that they are working
+   correctly together. In the case of Antrea, integration tests may create an
+   actual OVS bridge, which is why they depend on the OVS daemons running, and
+   are typically run inside a Docker image, even on Linux. Write integrations
+   tests when you require an actual OVS bridge or you need access to
+   platform-specific utilities.
+ * **end-to-end (e2e) tests**: these tests are located under [test/e2e] and are
+   also written using the [Go `testing`] package. Unlike the two previous test
+   categories, these assume that Antrea is running on an actual cluster and
+   require a kubeconfig file as well as SSH access to the cluster
+   Nodes. Instructions on how to run these tests, including how to setup a local
```

Comment:
These lines seem not wrapped strictly, e.g. the above line has enough space for "Node."

Reply:
It's funny, The auto-wrap in emacs refused to place Nodes on the same line as cluster. I had to do it manually...